scripts/install/functions.sh

```diff
@@ -95,6 +95,12 @@ function add_cdc {
     fi
 }
 
+function add_epd_visualisation {
+    if [ "$ADD_EPD_VISUALISATION" = true ] ; then
+        ng add --skip-confirmation @spartacus/epd-visualisation@${SPARTACUS_VERSION} --baseUrl ${EPD_VISUALISATION_BASE_URL} --interactive false
```

Comment:
It's my bad, I wrote a typo in my github comment....

Unfortunately this script does not install the lib successfully, because of the typo in the package name here `epd-visualiSation`(instead of `Z`). 

Btw. the same typo propagated also to other places, which would be also nice to fix, e.g. the shell function name, shell variable name, etc:
![image](https://user-images.githubusercontent.com/4001059/146586632-7f885b40-d15f-4a9d-bf05-9b79a8cd9407.png)

Reply:
FYI to run the installation script:
```shell
cd scripts/install
./run.sh install
```
(the script may run a few minutes)

Reply:
And prior to running the script you might want to set in the config.default.sh:
```shell
ADD_EPD_VISUALISATION=true
BACKEND_URL=<your occ backend>
BRANCH='epic/epd-visualization-maintenance'
```

Reply:
Sorry about that. I fixed this and I did have a working script, but the code got merged into the epic/epd-visualization branch but not this branch. I will fix it.

Reply:
Unfortunately, if I fix this typo locally (just to test if the script runs successfully), although the lib `@spartacus/epd-visualization` is successfully installed in customer's app, but the module code for this feature is not generated by the schematics - e.g. `epd-visualization-feature.module.ts` is not generated in the customer's folder `/app/spartacus/features` 🤔 

<img width="1480" alt="image" src="https://user-images.githubusercontent.com/4001059/146615292-22aa5097-0543-4f45-b006-4696ab703742.png">

Besides the long-running installation script, I also double-checked the code is not generated by schematics on the installation of the lib, by simply running manually the command `ng add @spartacus/epd-visualization` in the customer's app.
Prerequisites for that were:
- `ng remove @spartacus/epd-visualization` in customer's app
- I've opened the interactive script `tools/schematics/testing.ts` opened in another terminal window and chosen the interactive option "publish" (for more see https://github.com/SAP/spartacus/blob/develop/projects/schematics/README.md). 

   Note: this interactive script launches a fresh instance of verdaccio (and kills it when you terminate the process). 
   Note 2: The long-running installation script also kills its instance of verdaccio at the end of its execution. 
- I've run `ng add @spartacus/epd-visualization` in the already created customer's spartacus app

But unfortunately the result was only installing the lib (added to customer's package.json), but no additional code was generated by schematics in the customer's app. See logs in customer's app:
<img width="1119" alt="image" src="https://user-images.githubusercontent.com/4001059/146617205-69e85276-9ac5-4c45-b623-4a17655aaf19.png">


--
FYI "a few commits ago" the schematics was successfuly generating the `epd-visualization-feature.module.ts` code in the customer's app. I was testing it that time, also using the interactive script `tools/schematics/testing.ts` (in our monorepo) + executing `ng add @spartacus/epd-visualization` (in customer's app)

Can you confirm the installation script is working correctly on you another branch? (the result should be the `../spartacus-latest` folder with the EPD module code generated in `/apps/csr/src/app/spartacus/features/epd-visualization/epd-visualization-feature.module.ts`

Reply:
I have merged the missing commit. I have also updated the default features list and added code to allow the baseSite to be configured.
I have tested the csr and ssr apps and everything looks ok to me.

Reply:
Thanks for providing the commit 👍 
I'm sorry, there must have been something wrong on my local setup.
I retested now and it works great!